docs/modules.md

```diff
@@ -13,6 +13,28 @@ Bookshelf is designed with modularity in mind. Features that serve a common purp
 ```{include} _templates/featured-modules.md
 ```
 
+## 📚 Glossary
+
+Here are some of the key terms used throughout the documentation.
+
+:::{list-table}
+*   - **`ata` functions**
```

Comment:
I guess it's at the top because you wanted it to be in alphabetical order, but I think it would be better to place it after "Module."